code/modules/cargo/packs.dm

```diff
@@ -1873,6 +1968,16 @@
 	crate_name = "janitorial cart crate"
 	crate_type = /obj/structure/closet/crate/large
 
+/datum/supply_pack/service/janitor/janicart
+	name = "Janicart Crate"
+	desc = "You'd better not have wrecked the last one joyriding."
+	cost = 20000
+	access_budget = ACCESS_JANITOR
+	contains = list(/obj/vehicle/ridden/janicart,
```

Comment:
This isn't an issue for here, but I hate how the "pimpin' ride" and the "janitorial cart" share very similar typepath names.

Reply:
It's funny because that fact actually caused me some issues and confusion in making this and I had to clean it up a bit lol